packages/core/forms/src/locales/en.json

```
@@ -1,6 +1,8 @@
 {
   "general": {
-    "packageName": "forms"
+    "packageName": "forms",
+    "disable_global_radio": "The plugin is set to 'Scoped' by default when configured under a {scope}. To configure a global plugin, navigate to the plugin list page.",
```

How about adding `gloabl`?
```suggestion
    "disable_global_radio": "The plugin is set to 'Scoped' by default when configured under a {scope}. To configure a global plugin, navigate to the global plugin list page.",
```